### PR TITLE
[WNMGDS-2503] Add a draft of the release 8 blog post

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

Release 8.0 blog post

[Demo here](https://cmsgov.github.io/design-system/branch/pwolfert/release-8-blog-post/blog/release-8.0?theme=core)

### CODE REVIEW

File: packages/docs/content/blog/release-8.0.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Release 8.0
+date: 2023-09-18
+intro: Normalizing CSS, standardizing spacing & colors, a new Dropdown, and more
```
Comment (zarahzachz) [inline packages/docs/content/blog/release-8.0.mdx:4]: we don't talk about normalizing css do we?

File: packages/docs/content/blog/release-8.0.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+---
+
+import hcHeaderImg from '../../src/images/hc-header-banner.png';
+
+We know that Open Enrollment is just around the corner for both Medicare.gov and HealthCare.gov teams, so this update is intentionally light. We don’t expect most teams to have to change anything in this major release. We did mark a few breaking changes because they are either “breaking” in principal or have the potential to cause unexpected changes. We lumped those changes into a major release to make sure teams are paying attention to the upgrade. We hope this release gives teams more flexibility and at the same time helps them create more consistent experiences as we head into fall.
```
Comment (brentschneider) [inline packages/docs/content/blog/release-8.0.mdx:9]: "breaking" in principle

File: packages/docs/content/blog/release-8.0.mdx (hunk 1 above)
Comment (pwolfert) [reply packages/docs/content/blog/release-8.0.mdx:4]: Ah ha, no

File: packages/docs/content/blog/release-8.0.mdx (hunk 2 above)
Comment (pwolfert) [reply packages/docs/content/blog/release-8.0.mdx:9]: nice catch

File: packages/docs/content/blog/release-8.0.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+<div className="ds-u-measure--wide ds-u-text-align--center ds-u-margin-top--3">
+  <img src={hcHeaderImg} width={685} className="ds-u-display--inline-block" alt="HealthCare.gov header with the USA banner above it" />
+</div>
+
+To make HealthCare.gov compliant with government standards and increase trust with consumers, its shared header component will now have the [USA Banner](/components/usa-banner/) built into it. To help coordinate a synchronized release of this banner on all applications at once, we’ve included a prop that teams can use to show or hide it.
+
+### The Medicare.gov footer says goodbye to its glossary link
+
+Medicare.gov is discontinuing their glossary page, so we’re removing the link from the footer. Teams that implement their own footer are responsible for removing it in their own applications.
```
Comment (brentschneider) [inline packages/docs/content/blog/release-8.0.mdx:23]: suggesting _its_ instead of _their_

File: packages/docs/content/blog/release-8.0.mdx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+### Improved VoiceOver + Safari support for Autocomplete and Dropdown
+
+VoiceOver support for our Autocomplete and Dropdown components has not been as robust as it could be. The method used to fill in VoiceOver’s lack of standards compliance has been somewhat unreliable and leaves something to be desired in the user experience. In order to remedy this, we’ve moved away from using the [Downshift](https://www.downshift-js.com/) library to a solution that is better supported by VoiceOver while maintaining strong support from other screen readers. More details can be found in our code release notes.
+
+## What's on deck
+
+### Piloting our web components
+
+In the previous release, we started offering an experimental subset of our components as framework-agnostic web components. In the months leading up to the new year, we will be piloting these components with a couple teams. We'll learn and iterate on them, and we'll also apply what we've learned to the next batch of web components that we make.
```
Comment (brentschneider) [inline packages/docs/content/blog/release-8.0.mdx:35]: couple _of_ teams

File: packages/docs/content/blog/release-8.0.mdx (hunk 3 above)
Comment (pwolfert) [reply packages/docs/content/blog/release-8.0.mdx:23]: What would change to its?

Comment (brentschneider) [reply packages/docs/content/blog/release-8.0.mdx:23]: Medicare.gov is discontinuing its glossary page, so we’re removing the link from the footer. Teams that implement their own footer are responsible for removing it in their own applications.

Comment (pwolfert) [reply packages/docs/content/blog/release-8.0.mdx:23]: Oh, yeah, that makes sense

**Review by zarahzachz — APPROVED** [review]

**Review by brentschneider — APPROVED** [review]
Awesome, looking good!

**Pull request merged**